hoomd/write/table.py

```diff
@@ -208,12 +208,16 @@ def __init__(self,
 
         # internal variables that are not part of the state.
         # Ensure that only scalar and potentially string are set for the logger
-        if (LoggerCategories.scalar not in logger.categories
-                or logger.categories & self._invalid_logger_categories
-                !=  # noqa: W504 (yapf formats this incorrectly
-                LoggerCategories.NONE):
+        if LoggerCategories.scalar not in logger.categories:
```

Comment:
Why not accept a logger that only takes `string` categories?

Reply:
I'm not sure - I retained the original behavior (that required scalar categories to be set) but I don't see why a string-only logger wouldn't work.  @b-butler, do you have any insight into why it was initially set up this way? If you don't recall any issues, I will refine the logic to allow string-only loggers.

Reply:
We should. I just didn't originally for who knows why. @janbridley please change this here to fix my past mistake.

Reply:
Done! See [ec7b847](https://github.com/glotzerlab/hoomd-blue/pull/1519/commits/ec7b847767abe1ae6a363e944fd1f7539e966ca1)